### PR TITLE
Add integration tests for `validate` and `generate-schema`

### PR DESCRIPTION
**Pull request by andreafalzetti**
Fixes: DEV-811

In this PR we're adding integration tests for the 2 new commands: `validate` and `generate-schema`

### CODE REVIEW

**Review by teresaromero — APPROVED** [review]

File: integration/validate/manifests/valid-gcp-cloud-credentials.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+          gcloud storage buckets create "gs://cloud-cred-demo-${OKTETO_NAMESPACE}"
+        else
+          echo "Bucket gs://cloud-cred-demo-${OKTETO_NAMESPACE} already exists. Skipping creation."
+        fi
+
+    - name: Create GCP SA
+      command: |
+        set -e
+        saName="dev-env-${OKTETO_NAMESPACE}"
+        gcpProject="arsh-358508"
```
Comment (ifbyol) [inline integration/validate/manifests/valid-gcp-cloud-credentials.yml:21]:
```suggestion
        gcpProject="gcpProject"
```

File: integration/validate/manifests/valid-gcp-cloud-credentials.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        gcloud projects add-iam-policy-binding $gcpProject \
+        --member="serviceAccount:$saName@$gcpProject.iam.gserviceaccount.com" \
+        --role="roles/storage.admin"
+        
+        gcloud iam service-accounts add-iam-policy-binding $saName@$gcpProject.iam.gserviceaccount.com \
+        --role="roles/iam.workloadIdentityUser" \
+        --member="serviceAccount:$gcpProject.svc.id.goog[${OKTETO_NAMESPACE}/$saName]"
+
+    - name: Deploy the app
+      command: helm upgrade --install gcp-app gcp-app --set bucket="cloud-cred-demo-${OKTETO_NAMESPACE}" --set serviceAccountName="dev-env-${OKTETO_NAMESPACE}" --set gcpProject="arsh-358508" --set image=${OKTETO_BUILD_SERVER_IMAGE}
```
Comment (ifbyol) [inline integration/validate/manifests/valid-gcp-cloud-credentials.yml:33]:
```suggestion
      command: helm upgrade --install gcp-app gcp-app --set bucket="cloud-cred-demo-${OKTETO_NAMESPACE}" --set serviceAccountName="dev-env-${OKTETO_NAMESPACE}" --set gcpProject=gcpProject" --set image=${OKTETO_BUILD_SERVER_IMAGE}
```

File: integration/validate/manifests/valid-gcp-cloud-credentials.yml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+          gcloud storage buckets update $bucket --uniform-bucket-level-access
+          gcloud storage rm -r $bucket -q
+        else
+          echo "Bucket gs://cloud-cred-demo-${OKTETO_NAMESPACE} does not exist. Skipping deletion."
+        fi
+
+    - name: Delete GCP SA
+      command: |
+        saName="dev-env-${OKTETO_NAMESPACE}"
+        gcpProject="arsh-358508"
```
Comment (ifbyol) [inline integration/validate/manifests/valid-gcp-cloud-credentials.yml:52]:
```suggestion
        gcpProject="gcpProject"
```

**Review by ifbyol — APPROVED** [review]
Added a few inline comments to change some project name, but, looks good

**Review by jLopezbarb — APPROVED** [review]

**Pull request merged**